test/acceptance/Dockerfile

```diff
@@ -7,9 +7,6 @@ ARG PARENT_VERSION
 LABEL uk.gov.defra.ffc.parent-image=defradigital/node-development:${PARENT_VERSION}
 ARG PORT_DEBUG
 EXPOSE ${PORT_DEBUG}
-USER root
-RUN apk add --no-cache libstdc++ procps
-USER node
 COPY --chown=node:node package*.json ./
```

Comment:
Don't think we actually need to install this binary when using the npm package so can remove this